src/utils/url.js

```diff
@@ -369,3 +370,14 @@ export function isValidUrl (url) {
 
     return parsedUrl.protocol === 'file:' || parsedUrl.hostname && (!parsedUrl.port || isValidPort(parsedUrl.port));
 }
+
+export function ensureOriginTrailingSlash (url) {
+    // NOTE: If you request an url containing only port, host and protocol
+    // then browser adds the trailing slash to window.location.href.
```

Comment:
Change
`...then browser adds the trailing slash to window.location.href`.
to 
`...then browser adds the trailing slash itself`